bugwarrior/services/jira.py

```diff
@@ -290,16 +290,13 @@ def validate_config(cls, config, target):
     def annotations(self, issue, issue_obj):
         comments = self.jira.comments(issue.key)
```

Comment:
Seems like the CI build is failing now when there are no comments, since comments = None. Could we fix it by setting this with:

```
comments = self.jira.comments(issue.key) or []
```

?